Server/Plugins/APIDump/Hooks/OnPawnFell.lua

```diff
@@ -0,0 +1,25 @@
+return
+{
+	HOOK_PAWN_FELL =
+	{
+		CalledWhen = "A pawn has fallen onto a block",
+		DefaultFnName = "OnPawnFell",  -- also used as pagename
+		Desc = [[
+			This hook is called whenever a cPawn has fallen onto a block.
+		]],
+		Params =
+		{
+			{ Name = "Player", Type = "{{cPlayer}}", Notes = "The player from whom the packet was received" },
+			{ Name = "Animation", Type = "number", Notes = "The kind of animation" },
```

Comment:
This doesn't match up with how the hook is actually called, with a `Damage` rather than an `Animation`.